Persistence/Repositories/AttachmentStorage.cs

```diff
@@ -50,3 +48,11 @@ public void Delete(string blobUri, CancellationToken cancellationToken)
         throw new NotImplementedException();
     }
 }
+
+public interface IAttachmentStorage
```

Comment:
This interface should remain in the Domain project. See https://stackoverflow.com/a/72399927 for reasons why